controllers/tasks.js

```diff
@@ -123,10 +123,33 @@ const updateTask = async (req, res) => {
   }
 }
 
+/**
+ * Updates self task status
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const updateTaskStatus = async (req, res) => {
+  try {
+    const taskId = req.params.id
+    const { id: userId } = req.userData
+    const task = await tasks.fetchSelfTask(taskId, userId)
+
+    if (task.taskIdNotFound) return res.boom.notFound("Task doesn't exist")
+    if (task.notAssignedToYou) return res.boom.forbidden('This task is not assigned to you')
```

Comment:
Inconsistency in using quotes. Let's just stick to single quotes. You can escape the apostrophe, check how it's done in other routes ;)
Hint: you'll find a slash ;)